library/kubevirt_vm.py

```diff
@@ -98,32 +104,180 @@
 RETURN = ''' # '''
 
 from ansible.module_utils.basic import AnsibleModule
-import base64
+from ansible import errors
 from kubernetes import client, config
-import os
+from kubernetes.client.rest import ApiException
+import base64
 import time
 import yaml
 
 DOMAIN = "kubevirt.io"
 VERSION = 'v1alpha1'
-REGISTRYDISKS = ['kubevirt/alpine-registry-disk-demo', 'kubevirt/cirros-registry-disk-demo', 'kubevirt/fedora-cloud-registry-disk-demo']
+REGISTRYDISKS = [
+    'kubevirt/alpine-registry-disk-demo',
+    'kubevirt/cirros-registry-disk-demo',
+    'kubevirt/fedora-cloud-registry-disk-demo']
+
+
+def build_vm_definition(params):
+    '''returns a dict() containing a VM definition based on params.'''
```

Comment:
Imperative mood, present tense in all docstring please, i.e. `return, delete, remove, update, build, validate, etc`. No 's'. Thanks.